### DISCUSSION TITLE
Use other affiliate than Kobo

### DISCUSSION THREAD

**Discussion by frederic-blanc** [subdavis/kobo-book-downloader#82] [Ideas]
Hello,

Is there some solution to provide more affiliate than just the Kobo one? the Fnac one in my case.
With the old version I change just the Kobo.Affiliate  value in the python file, and just rebuild the container, but sadly it does not work anymore.
And the refresh token in my kododl.json seems to old to be used one more time.

**Comment by subdavis** [comment]
From the README:

> NOTE: You must have a kobo email login for this tool to work (you can't use an external provider like Google or Facebook). However, you can create a NEW kobo account and link it with your existing account on the user profile page. Go to My Account -> Account Settings to link your new kobo login.

**Comment by frederic-blanc** [comment]
I will try this, but it seems that I cannot use the same email for the two accounts.
thank

**Comment by subdavis** [comment]
It's quite possible that, with the new web auth method, other accounts work.  I haven't tried, but someone should check and report back.

**Comment by joren485** [comment]
I can confirm that logging in using an affiliate (bol.com in my case) works with the new authentication method.
